modules/renter/persist.go

```diff
@@ -72,3 +88,72 @@ func (r *Renter) load() error {
 	}
 	return nil
 }
+
+// ShareFiles saves a '.sia' file that can be shared with others, enabling them
+// to download the file you are sharing. It creates a Sia equivalent of a
+// '.torrent'.
+func (r *Renter) ShareFiles(nicknames []string, sharedest string) error {
+	lockID := r.mu.RLock()
+	defer r.mu.RUnlock(lockID)
+
+	if len(nicknames) == 0 {
+		return ErrNoNicknames
+	}
+	// Suffix enforcement is not really necessary, but I want explicit
+	// enforcement of the suffix until people are used to seeing '.sia' files.
+	if !strings.HasSuffix(sharedest, ShareExtension) {
+		return ErrNonShareSuffix
+	}
```

Comment:
`filepath.Ext` might be clearer here, and you're already including `filepath` anyway